README.md

```diff
@@ -116,6 +116,20 @@ rather than a cluster:
 polaris --audit --audit-path ./deploy/
 ```
 
+##### Exit Codes for Audits
+<dl>
+  <dt>Exit 0</dt>
+  <dd>Successful exit code</dd>
+  <dt>Exit 1</dt>
+  <dd>Could not run audit, or application had a failure while running.</dd>
```

Comment:
This might be better described as "unexpected error". We seem to use it like an HTTP `500`